### PR TITLE
updated second readme for prrr

### PR DESCRIPTION
**Pull request by imsolost**

### CODE REVIEW

**Review by jrobcodes — CHANGES_REQUESTED** [review]
Please refine with more explicit specifications (a reviewer should be able to answer the binary question, "Is this specification completed"), as well as a quality rubric (check other repos, but a good quality rubric should include testing, linting for code quality, etc.).

**Comment by Anasauce** [comment]
Hey there looks like you addressed @jrob8577 's concerns, good job! Next time you can do this on the same branch as you original PR and then commit and push changes and re-submit your PR for review :)

**Review by jrobcodes — CHANGES_REQUESTED** [review]
The quality rubric is missing a test suite.

File: readme2.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- [ ] Game saves win loss record after versus games
+- [ ] Add exploration mode where players can choose different paths by going up or down
+- [ ] Database saves a record of path so future players can try to find paths other players have taken for a bonus.
+- [ ] "Chimera mode" where various power ups change both your bird's appearance and how it interacts with obstacles
+- [ ] At end of "Chimera Mode" the game gives the option to save an image of your final bird with all powerups
+- [ ] Add shooter mode with enemy birds, weapons, and powerups to enhance your bird
+- [ ] Test backend and testable front end functions with chai
+
+## Stretch Goals
+- [ ] Test front end with selenium
```
Comment (jrobcodes) [inline readme2.md]: If you're using phaser, you're going to have a hard time testing the front end with selenium (or it's going to simply be a test that check is a file loads, which isn't a very meaningful test).  I strongly recommend the use of a unit testing framework for backend logic.

**Review by jrobcodes — CHANGES_REQUESTED** [review]
More comments - please test.

I'm still a little leary of this as a project.  I think it will be fun, but I'm questioning whether or not incorporating phaser into a web application is really moving you towards the goal of being a web developer...  This feels like a goal that should be attempted when you have enough experience to understand why you would choose a library like phaser, and how such a library works.  What problems is phaser solving? Why not implement it directly with websockets, js, and canvases? etc.

**Comment by mantinone** [comment]
@jrob8577 

We listed unit testing with Chai as one of our main specs, as is done in several of the Core Goals.  Should we also mention it in the quality rubric?

It sounds like your recommendation would be to keep the same or similar specs, but without using Phaser.  Do you agree that that would be a good path to take?  

I have a pretty good idea of the range of problems a game engine is created to solve (and which specific subset of those problems Phaser seems to address), but it would be good exercise to learn how to handle those issues ourselves in the context of a web app.  The primary problems it deals with in the context of this project are rendering and animating images in a canvas, time and update-loops for simulation, and physics sim and collision (collision can be difficult, though the physics and collision in a Flappy Bird game are pretty simple).  Any of the network logic would be on us to figure out.  Phaser doesn't deal with that.

**Comment by jrobcodes** [comment]
Thanks for pointing that out, I always assume that tests will be in the quality portion of the spec, so my eyes must have scanned over it.  😄  

Yeah, my recommendation is to not user Phaser.  I don't see the value of it in the context of being a kick ass web developer (despite the fun it would no doubt be to tackle a new framework).  A lot of the problems it solves are problems you should be grappling with, and understanding the technologies that are used to implement the solution.  I fear that our focus framework centric projects (including React!) is obscuring some of this important, underlying stuff.  

But, that's just a recommendation.  If you think there's important stuff to learn (like network logic, which I agree is important) outside of phaser, then I could see a case being made for being able to ignore the other stuff you're not interested in by applying phaser to that stuff, and focusing on other pieces of the tech.

**Comment by mantinone** [comment]
@jrob8577 Specs are updated.

We've decided to pursue a focus on user auth, network play and syncing data between users in real time.  We also listed implementing the basic game without Phaser as stretch goals.

**Review by nicosesma — COMMENTED** [review]
👍

**Comment by mantinone** [comment]
I am merging this for the simplicity of git management.  It seems like the specs are probably in an okay place and we are making progress.  We can make changes later if necessary.

**Pull request merged**